### ISSUE TITLE
Required dependencie missing when installing the library

### ISSUE THREAD

**Issue by Henrique-Coder** [daijro/hrequests#15]
**OS**: Windows 11.0
**Python version**: 3.11.4

When installing the "hrequests" package using the pip command and executing some related code, some specific errors are observed, such as:
- ModuleNotFoundError: No module named 'bs4'
- ModuleNotFoundError: No module named 'BeautifulSoup'

It is noticeable that, even when using only the "get" function provided by the "hrequests" package, the "bs4" library is requested, and even after trying to install the "hrequests[all]" package, the same errors persist. However, when the "bs4" library is installed manually, the code works without problems. This is a rather inconvenient problem to solve every time I use this library in another project.

**Comment by daijro** [comment]
Hello,
Could you comment the traceback log that returns with this error?

Thanks!

**Comment by Henrique-Coder** [comment]
Okay, thanks. If you still need the error logs, here they are (I forgot to send them before):

```
C:\Users\*****\Downloads>py
Python 3.11.4 (tags/v3.11.4:d2340ef, Jun  7 2023, 05:45:37) [MSC v.1934 64 bit (AMD64)] on win32
Type "help", "copyright", "credits" or "license" for more information.
>>> from hrequests import get
WARNING: Please run `pip install hrequests[all]` for headless browsing support.
Traceback (most recent call last):
  File "C:\Users\*****\AppData\Roaming\Python\Python311\site-packages\lxml\html\soupparser.py", line 10, in <module>
    from bs4 import (
ModuleNotFoundError: No module named 'bs4'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "C:\Users\*****\AppData\Roaming\Python\Python311\site-packages\hrequests\__init__.py", line 19, in <module>
    from .parser import HTML
  File "C:\Users\*****\AppData\Roaming\Python\Python311\site-packages\hrequests\parser.py", line 13, in <module>
    from lxml.html.soupparser import fromstring as soup_parse
  File "C:\Users\*****\AppData\Roaming\Python\Python311\site-packages\lxml\html\soupparser.py", line 15, in <module>
    from BeautifulSoup import (
ModuleNotFoundError: No module named 'BeautifulSoup'
>>>
```

**Comment by daijro** [comment]
This has been fixed in the v0.8.0 beta, and should be rolled out with the next release.
